polyply/tests/test_apply_modifications.py

```diff
@@ -134,25 +144,32 @@ def test_apply_mod(input_itp, molname, expected, caplog):
                                                        meta=interaction.meta)
                             assert _interaction in meta_mol.molecule.interactions[interaction_type]
 
-@pytest.mark.parametrize('modifications, expected',
+@pytest.mark.parametrize('modifications, expected, text',
      (
              (
                  [['A1', 'N-ter']],
-                 True
+                 True,
+                 None
```

Comment:
I'm about 95% certain it's meant to be testing this line, in that the meta_mol force field doesn't have any modifications present. I guess it's kinda redundant because the next test ends up actually testing the same behaviour too, I'm not quite sure why I thought it was important as well.

https://github.com/marrink-lab/polyply_1.0/pull/385/files#diff-8eedddf55f9448422f33f35d28b9802477875269f38ab760b26e9c38600ee6bbL63